index.html

```diff
@@ -2035,9 +2030,36 @@ <h3>Securing Mechanisms</h3>
 mutually exclusive.
         </p>
         <p>
-<a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST be secured by at least
-one securing method.
+<a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST be
+secured by at least one securing mechanism.
+        </p>
+        <p>
```

Comment:
```suggestion
        </p>
        <p>
Verifiers MUST validate and trust the specific securing method, understanding its security implications. This is important because security guarantees MAY vary depending on the type of security mechanisms within a securing class and between securing mechanism classes.
        </p>
        <p>        
```

Reply:
Hrm, so this section is about normative requirements for securing mechanism specifications. 

The text above is about normative requirements for the Verifier role. I have raised issue #1386 to track your concern above.